build/build-sdk-images/restapi/sdktest.sh

```diff
@@ -17,8 +17,6 @@
 set -ex
 GO111MODULE=on
 cd /go/src/agones.dev/agones/test/sdk/restapi
-# swagger gen has a bug wherein it doesn't generate the file, so we're providing it by hand
-cp ./model_xstreamdefinitionssdkgameserver.go.nolint ./swagger/model_xstreamdefinitionssdkgameserver.go
```

Comment:
Removed `model_xstreamdefinitionssdkgameserver.go.nolint` file because `protoc-gen-openapiv2` doesn't support `x-stream-definitions`.
Ref: https://github.com/grpc-ecosystem/grpc-gateway/pull/1112